.ci/checker-framework.groovy

```diff
@@ -113,12 +113,24 @@ private static int checkCheckerFrameworkReport(final String profile) {
 private static List<List<String>> getCheckerFrameworkErrors(final String profile) {
     final List<String> checkerFrameworkLines = new ArrayList<>()
     final String command = "mvn -e --no-transfer-progress clean compile -P${profile}"
-    final Process process = getOsSpecificCmd(command).execute()
-    process.in.eachLine { final line ->
-        checkerFrameworkLines.add(line)
-        println(line)
+    final ProcessBuilder processBuilder = new ProcessBuilder(getOsSpecificCmd(command).split(' '))
+    processBuilder.redirectErrorStream(true)
+    final Process process = processBuilder.start()
+
+    BufferedReader reader = null
+    try {
+        reader = new BufferedReader(new InputStreamReader(process.inputStream))
+        String lineFromReader = reader.readLine()
+        while (lineFromReader != null) {
+            println(lineFromReader)
+            checkerFrameworkLines.add(lineFromReader)
+            lineFromReader = reader.readLine()
+        }
+        process.waitFor()
+    } finally {
+        reader.close()
```

Comment:
For writting code this way, you need to verify the reader isn't null, otherwise you will get a NPE.

Reply:
I thought about this but why would it be null? None of the constructors throw an exception.

Reply:
@rnveach Ping, let's get this done.